sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -594,17 +594,21 @@ private static void printVerdict(final long numberOfTestFailures, final File tes
         }
 
         if (numberOfTestFailures == 0) {
-            LOG.info(
-                    "Test run with {} Tests was completed successfully. No problems were found.",
-                    testRunObserver.getTotalNumberOfTestsRun());
+            if (testRunObserver.isInvalid()) {
```

Comment:
Move this out of the conditional block it is currently in, because whenever the test run is invalid the individual test results are irrelevant, because they can't be trusted in that case.

Reply:
done